ai-platform-unified/notebooks/notebook_template.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<p>I think this needs to be:</p><p> </p><pre class="ql-syntax" spellcheck="false">import&nbsp;os
if&nbsp;not&nbsp;os.getenv("AUTORUN"):
&nbsp;&nbsp;&nbsp;&nbsp;#&nbsp;Automatically&nbsp;restart&nbsp;kernel&nbsp;after&nbsp;installs
&nbsp;&nbsp;&nbsp;&nbsp;import&nbsp;IPython
&nbsp;&nbsp;&nbsp;&nbsp;app&nbsp;=&nbsp;IPython.Application.instance()
&nbsp;&nbsp;&nbsp;&nbsp;app.kernel.do_shutdown(True)
</pre><p> </p><p>Otherwise, the tests will fail.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/251/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done.